x/evm/types/command.go

```diff
@@ -162,12 +162,12 @@ func NewApproveContractCallWithMintCommand(
 }
 
 // DecodeParams returns the decoded parameters in the given command
-func (c Command) DecodeParams() (map[string]string, error) {
+func (m Command) DecodeParams() (map[string]string, error) {
 	params := make(map[string]string)
 
-	switch c.Type {
+	switch m.Type {
 	case COMMAND_TYPE_APPROVE_CONTRACT_CALL_WITH_MINT:
-		sourceChain, sourceAddress, contractAddress, payloadHash, symbol, amount, sourceTxID, sourceEventIndex := decodeApproveContractCallWithMintParams(c.Params)
+		sourceChain, sourceAddress, contractAddress, payloadHash, symbol, amount, sourceTxID, sourceEventIndex := DecodeApproveContractCallWithMintParams(m.Params)
```

Comment:
Same question about these.

Reply:
think the function is called in command_test,